api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/processor/MakePaymentMessageProcessor.java

```diff
@@ -40,4 +41,11 @@ public MessageType supportsMessageType() {
         return MAKE_PAYMENT;
     }
 
+    @Override
+    @Transactional(Transactional.TxType.REQUIRES_NEW)
+    public void processFailedMessage(Message message, FailureEvent failureEvent) {
```

Comment:
This (and its test) are identical in both payment processors so I was wondering whether they should have some shared inheritance. Any thoughts? At least it should be in another PR if that's the case so I've left it for now.